lesson12/index.html

```diff
@@ -10,8 +10,6 @@
     <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
-    <div id="js-loading">
-    </div>
     <div>
       <ul id="js-list"></ul>
       <button type="button" id="js-button">Click</button>
```

Comment:
I decided to make "js-loading" with javascript, so I deleted it.
This is so that the showloading () function can be reused.